tests/error/linear_errors/branch_use_field2.err

```diff
@@ -0,0 +1,7 @@
+Guppy compilation failed. Error in file $FILE:21
+
+19:    if b:
+20:        measure(s.q2)
+21:    return s
+              ^
+GuppyError: Field `s.q2` with linear type `qubit` was already used (at 20:16)
```

Comment:
Maybe "already used in some control flow paths"?

Reply:
Agreed, this would be better. However, we need to do some extra checks to determine if it's in "some" or "all" control-flow paths.

I'd prefer leaving this to another PR since it's not directly related to structs (the same error shows up if you use a regular variable instead of a struct access). See #306